### PR TITLE
Heart means heart

### PR DESCRIPTION
**Pull request by rjmk**
People often want to react 'heart' but our ❤️  means 'empathy', which is confusing. This makes a new heart react and deprecates empathy

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1207747300443387) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/lib/voting/votingSystems.tsx
Comment (rjmk) [inline packages/lesswrong/lib/voting/votingSystems.tsx:179]: @jimrandomh I believe we don't use this or emojiReactions anymore, right? Your name is on the blame so thought I'd check with you

**Comment by b0b3rt** [comment]
@rjmk I think that changing the names of existing reacts isn't backwards-compatible, since reacts that were left with the old name will now be orphaned and might break stuff when we try to do anything with it.  Changing the label might be safe?  Not sure.  Another option is to just make a new react with the new name but the same icon, and set the `deprecated` flag to `true` on the old one.

**Comment by rjmk** [comment]
Thanks, Robert. Seems likely

**Comment by rjmk** [comment]
[This](https://github.com/ForumMagnum/ForumMagnum/blob/94c57d67158b67f76cca02b2f6000ab1105e08d3/packages/lesswrong/lib/voting/reactions.tsx#L76) is a fairly confusing variable.

- Why is there no heart/empathy in it? 
- on line 229 it says `Here begins the list of deprecated reacts` but the following reacts aren't deprecated?
- A comment says `There are spaces between each person to help reason about how they'll be displayed in the two columns, two on each row.`, but I don't know what spaces are being talked about

**Comment by rjmk** [comment]
@b0b3rt in principle, I would rather do a migration than either having the name "empathy" lingering around or having some old "empathy" reactions lingering around. But I'm guessing that's pretty messy because IIRC we store vote information as basically JSON blobs in the db. Is that right?

**Comment by b0b3rt** [comment]
> @b0b3rt in principle, I would rather do a migration than either having the name "empathy" lingering around or having some old "empathy" reactions lingering around. But I'm guessing that's pretty messy because IIRC we store vote information as basically JSON blobs in the db. Is that right?

Yeah, we store it nested inside of a JSON field on the vote record, unfortunately.  If you want to make a general-purpose migration script for renaming reacts I wouldn't mind, but it might not be the best use of time.

File: packages/lesswrong/lib/voting/votingSystems.tsx
Comment (oetherington) [reply packages/lesswrong/lib/voting/votingSystems.tsx:179]: EA Forum still has posts using these voting systems

**Comment by Will-Howard** [comment]
@rjmk Are you waiting on a review from me? It looks good apart from the issue Ollie mentioned about the forum still using those old voting systems

File: packages/lesswrong/lib/voting/votingSystems.tsx
Comment (rjmk) [reply packages/lesswrong/lib/voting/votingSystems.tsx]: Cool, I've put them back in

**Review by darkruby501 — APPROVED** [review]

**Pull request merged**
